src/oic/utils/keyio.py

```diff
@@ -460,7 +464,7 @@ def add(self, issuer, url, **kwargs):
         if "/localhost:" in url or "/localhost/" in url:
             kc = self.keybundle_cls(source=url, verify_ssl=False, **kwargs)
```

Comment:
this should probably get the timeout too.

Reply:
I am not sure about this. Although looking at the condition this can reach the network if the url is poorly chosen, so it might be safer to add it.